boards/ti/lp_mspm0g3507/doc/index.rst

```diff
@@ -0,0 +1,160 @@
+
+  * One 12-bit 1-Msps DAC.
+
+  * Three high speed comparators with built-in reference DACs.
+
+  * Two zero-drift zero-crossover op-amps with programmable gain.
+
+* Digital.
+
+  * Two 16-bit advanced control timers.
+
+  * Five general-purpose timers.
+
+    * One 16-bit general-purpose timer for QEI interface.
+
+    * One 32-bit high resolution general-purpose timer.
+
+    * Two 16-bit timers with deadband support and up to 12 PWM Channels.
+
+  * Two windowed-watchdog timers.
+
+  * One RTC with alarm and calendar modes.
+
+* Data Integrity and Encryption.
+
+  * One AES HW accelerator capable of CTR, CBC, and ECB modes.
+
+  * One Cyclic Redundancy Check (CRC) accelerator.
+
+  * One True Random Number Generator (TRNG).
+
+* Communication.
+
+  * Four UARTs, one with support for advanced modes such as LIN and Manchester.
+
+  * Two I2C supporting SMBUS/PMBUS and speeds up to FM+ (1Mbits/s).
+
+  * Two SPI, one with max speed 32Mbits/s.
+
+  * One CAN interface supporting CAN 2.0 A or B and CAN-FD.
+
+.. image:: img/lp_mspm0g3507.webp
+     :align: center
+     :alt: MSPM0G3507 LaunchPad development board
+
+Zephyr uses the ``lp_mspm0g3507`` board for building LP_MSPM0G3507
+
+Features:
+*********
+
+- Onboard XDS110 debug probe
+- EnergyTrace technology available for ultra-low-power debugging
+- 2 buttons, 1 LED and 1 RGB LED for user interaction
+- Temperature sensor circuit
+- Light sensor circuit
+- External OPA2365 (default buffer mode) for ADC (up to 4 Msps) evaluation
+- Onboard 32.768-kHz and 40-MHz crystals
+- RC filter for ADC input (unpopulated by default)
+
+Details on the MSPM0G3507 LaunchPad can be found on the `TI LP_MSPM0G3507 Product Page`_.
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Building and Flashing
+*********************
+
+Building
+========
+
+Follow the :ref:`getting_started` instructions for Zephyr application development.
+
+For example, to build the blinky application for the MSPM0G3507 LaunchPad:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: lp_mspm0g3507
+   :goals: build
+
+The resulting ``zephyr.bin`` binary in the build directory can be flashed onto
+MSPM0G3507 LaunchPad using the steps mentioned below.
+
+Flashing
+========
+
+Open OCD is used to program the flash memory on the devices. It may be necessary in
+earlier versions to use a branch of open OCD onto the device.
+
+Before OpenOCD is public, one can clone `This Repo <https://github.com/openocd-org/openocd.git>`_,
+and then this can be built with
+
+```
+cd <cloned_OPENOCD_dir>
+./bootstrap (when building from the git repository)
+./configure
+make
+sudo make install
+```
+
+Then after the build, it is possible to flash the device by passing additional arguments to the flash command
+
+```
+west flash --openocd <path to cloned dir>/src/openocd --openocd-search <path to cloned dir>/tcl
+```
+
+Flashing using JLINK
+
+```
+west flash --runner=jlink
+```
```

Comment:
Just noticed this -- this is not correct for ReStructuredText format, please use `.. code-block ::`